lib/business_calendar/version.rb

```diff
@@ -1,3 +1,3 @@
 module BusinessCalendar
-  VERSION = "1.1.0"
+  VERSION = "1.1.1"
```

Comment:
might want to make this a major version change, since we're significantly changing most of the default UK holiday behavior